translation/es-ES.toml

```diff
@@ -1,27 +1,35 @@
 name = "Español" # Name of language
 direction = "ltr" # Change to "rtl" for right to left languages
-version = "21"
+version = "23"
 maintainer = "xa31"
 
 [header]
 home = "Inicio"
```

Comment:
The value of this in update 22 was changed from "Home" to "Infinite Chess" in the english TOML. Does this need to change?

Reply:
Oh, sorry, I missed that.